lib/cea/cea_decoder.js

```diff
@@ -245,6 +291,62 @@ shaka.cea.CeaDecoder = class {
     return parsedClosedCaption;
   }
 
+  /**
+   * Decodes a CEA-708 DTVCC packet based on ANSI/CTA-708-E.
+   * @param {shaka.cea.DtvccPacket} dtvccPacket
+   * @return {!Array<!shaka.cea.ICaptionDecoder.ClosedCaption>}
+   * @private
+   */
+  decodeCea708_(dtvccPacket) {
+    const parsedClosedCaptions = [];
+    try {
+      while (dtvccPacket.hasMoreData()) {
+        // Process a service block.
+        const serviceBlockHeader = dtvccPacket.readBlock().byte;
```

Comment:
I find this line confusing.  You read a "block" from the packet, but throw it away except for a "byte" in it.  If that byte is a "block header", how did we "read block" already?

Let's discuss in our meeting.